code/modules/smithing/anvil.dm

```diff
@@ -300,7 +300,7 @@
 	desc = "A solid steel anvil with a stamped bull on it."
 	icon_state = "legvil"
 	anvilquality = 1
-	itemqualitymax = 8
+	itemqualitymax = 10
```

Comment:
This is an absurd buff. It doesn't do what you think it does, if you think this is a minor change.